zetta_utils/mazepa_layer_processing/common/volumetric_apply_flow.py

```diff
@@ -620,6 +621,43 @@ def flow(  # pylint:disable=too-many-branches, too-many-statements
             yield mazepa.Dependency()
             clear_cache(dst_temp)
             delete_if_local(dst_temp)
+        elif self.processing_blend_mode == "defer":
+            assert dst is not None
+            reduction_chunker = VolumetricIndexChunker(
+                chunk_size=dst.backend.get_chunk_size(self.dst_resolution),
+                resolution=self.dst_resolution,
+                max_superchunk_size=self.max_reduction_chunk_size_final,
+            )
+            logger.info(
+                f"Breaking {idx} into reduction chunks with checkerboarding"
+                f" with {reduction_chunker}. Processing chunks will use the padded index"
+                f" {idx.padded(self.roi_crop_pad)} and"
+                f" be chunked with {self.processing_chunker}.\n"
+                f"Note that because blending is deferred, {dst} will NOT "
+                f"have the final output."
+            )
+            stride_start_offset = dst.backend.get_voxel_offset(self.dst_resolution)
+            stride_start_offset_in_unit = stride_start_offset * self.dst_resolution
+            red_chunks = reduction_chunker(
+                idx, mode="exact", stride_start_offset_in_unit=stride_start_offset_in_unit
+            )
+            red_shape = reduction_chunker.get_shape(
+                idx, mode="exact", stride_start_offset_in_unit=stride_start_offset_in_unit
+            )
+            (
+                tasks,
+                red_chunks_task_idxs,
+                red_chunks_temps,
+                dst_temps,
+            ) = self.make_tasks_with_checkerboarding(
+                idx.padded(self.roi_crop_pad), red_chunks, red_shape, dst, op_kwargs
+            )
+            logger.info(
+                "Writing to intermediate destinations:\n"
+                f" Submitting {len(tasks)} processing tasks from operation {self.op}."
+            )
+            yield tasks
+            yield mazepa.Dependency()
         # case with checkerboarding
```

Comment:
This should be moved to above the last case, and modified to `# cases with checkerboarding`

Reply:
(The comment on #572 should also be plural - my bad.)